index.bs

```diff
@@ -1766,14 +1767,14 @@ The semantics of the fields in the [=attestation object=] are as follows:
 
 #### Privacy #### {#sec-attestation-privacy}
 
-Attestation keys may be used to track users or link various online identities of the same user together. This may be mitigated
+Attestation keys MAY be used to track users or link various online identities of the same user together. This MAY be mitigated
 in several ways, including:
 
-- A WebAuthn [=authenticator=] manufacturer may choose to ship all of their devices with the same (or a fixed number of)
+- A WebAuthn [=authenticator=] manufacturer MAY choose to ship all of their devices with the same (or a fixed number of)
     attestation key(s) (called [=Basic Attestation=]). This will anonymize the user at the risk of not being able to revoke a
-    particular attestation key should its WebAuthn Authenticator be compromised.
+    particular attestation key if its WebAuthn Authenticator be compromised.
 
-- A WebAuthn Authenticator may be capable of dynamically generating different attestation keys (and requesting related
+- A WebAuthn Authenticator MAY be capable of dynamically generating different attestation keys (and requesting related
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
updated this section to not have normative text.